src/OrchardCore.Modules/OrchardCore.Facebook/Settings/FacebookLoginSettings.cs

```diff
@@ -1,7 +1,9 @@
+using Microsoft.AspNetCore.Http;
+
 namespace OrchardCore.Facebook.Settings
 {
     public class FacebookLoginSettings
     {
-        public string CallbackPath { get; set; }
+        public PathString CallbackPath { get; set; }
```

Comment:
Shouldn't we use `/signin-facebook` as the default value here as this is be what users want most of the time?